core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -529,7 +532,8 @@ case object ModelInsights {
     rawFeatures: Array[features.OPFeature],
     blacklistedFeatures: Array[features.OPFeature],
     blacklistedMapKeys: Map[String, Set[String]],
-    rawFeatureDistributions: Array[FeatureDistribution]
+    rawFeatureDistributions: Array[FeatureDistribution],
+    rawFeatureFilterResults: RawFeatureFilterResults = RawFeatureFilterResults()
```

Comment:
@clin-projects is the default value for `rawFeatureFilterResults ` provided here for backward compatibility?

Reply:
@clin-projects also, `RawFeatureFilterResults` already includes raw feature distributions, why do we have both here?

Reply:
Thanks @tovbinm!

Actually, both are choices for backward compatibility, so that we can continue to load old workflows. Currently if we use `loadJson()` from `OpWorkflowModelReader` (and any other load workflow function), we will be doing it with a Json that has `rawFeatureDistributions` but not `rawFeatureFilterResults`. This redundancy (and empty default value) allows us to load in older workflows that do not have `rawFeatureFilterResults`.

A next-step addition will be to change `loadJson()` to:
(1) check if there is `RawFeatureFilterResults`, and
(2) call `setRawFeatureFilterResults`.
In this way, loading a workflow also loads in the RFF results. However, at a minimum, we cannot do this until *after* we have implemented a way to output the RFF Results when we save the workflow.

The next part of this set of PRs will be to modify `ModelInsights` so that all RFF results are exposed

Reply:
Can you add a todo to that effect?

Reply:
Yes, I'll add a todo in `loadJson()`